core/domain/prod_validators_test.py

```diff
@@ -1453,6 +1449,7 @@ def setUp(self):
                 'property_name': 'title',
                 'new_value': 'New title'
             })], 'Changes.')
+        self.process_and_flush_pending_tasks()
```

Comment:
Ditto?

Reply:
The change was made after updating local branch(this was updated in master branch)